consoleme/lib/alb_auth.py

```diff
@@ -38,17 +39,25 @@ async def authenticate_user_by_alb_auth(request):
         config.get("get_user_by_aws_alb_auth_settings.jwt_email_key", "email")
     )
 
+    if not email:
+        raise UnableToAuthenticate("Unable to determine user from ID Token")
+
     # Step 4: Parse the Access Token
     # User has already passed ALB auth and successfully authenticated
-    access_token_jwt = jwt.decode(encoded_claims_jwt, pub_key, verify=False)
-    groups = access_token_jwt.get(
-        config.get("get_user_by_aws_alb_auth_settings.jwt_groups_key", "groups")
-    )
-    # Step 5: Verify the access token.
-    validate_token(
-        encoded_claims_jwt,
-        access_token_jwt["iss"],
-        access_token_jwt["aud"],
-        access_token_jwt["cid"],
-    )
+    try:
+        access_token_jwt = jwt.decode(encoded_claims_jwt, pub_key, verify=False)
+        groups = access_token_jwt.get(
+            config.get("get_user_by_aws_alb_auth_settings.jwt_groups_key", "groups")
+        )
+        # Step 5: Verify the access token.
+        validate_token(
+            encoded_claims_jwt,
+            access_token_jwt["iss"],
+            access_token_jwt["aud"],
+            access_token_jwt["cid"],
+        )
+    except jwt.exceptions.DecodeError:
+        # Silently skip. This is usually due to a JWT that doesn't have JWT-parsable claims.
```

Comment:
I would recommend adding this to a log. Otherwise, people will be scratching their heads trying to figure out what's going on.

Reply:
I'll add a debug log. I was debating with myself whether it would be appropriate because for google oauth, we'd get a log for every invocation

Reply:
Fixed